ironic-scripts/configure-ironic.sh

```diff
@@ -8,6 +8,7 @@ export IRONIC_INSECURE=${IRONIC_INSECURE:-false}
 export IRONIC_INSPECTOR_CERT_FILE=/certs/ironic-inspector/tls.crt
 export IRONIC_INSPECTOR_CACERT_FILE=/certs/ca/ironic-inspector/tls.crt
 export IRONIC_INSPECTOR_INSECURE=${IRONIC_INSPECTOR_INSECURE:-$IRONIC_INSECURE}
+export RESTART_CONTAINER_CERTIFICATE_UPDATED=${RESTART_CONTAINER_CERTIFICATE_UPDATED:-"false"}
```

Comment:
It seems to me that rather than have an environment variable we could just check whether inotifywait exists, and assume that if the functionality was built into the container that we want to use it.

Reply:
I think the inotify-tools will always be baked into the image, but it is the user's choice to use it or not. That is why we need this environment variable.